app/Http/routes.php

```diff
@@ -76,6 +76,12 @@
         ]
     ]);
 
+    /*Routes for bookmarks*/
+    Route::group(['prefix' => 'bookmarks'], function () {
```

Comment:
Or maybe will be better if I use `topics/{topic}/bookmarks` url format? But in this way i can't check uniqueness of composite value topic_id-user_id in validation layer, and should do that in Controller.